src/qt/veil/addresseswidget.cpp

```diff
@@ -240,6 +243,16 @@ void AddressesWidget::handleAddressClicked(const QModelIndex &index){
         listView = ui->listAddresses;
         type = AddressTableModel::Receive;
         updatedIndex = proxyModel->mapToSource(index);
+        auto address = this->model->index(index.row(), AddressTableModel::Address, index);
```

Comment:
This needs to be the updatedIndex.

Reply:
Updated the line.  both index.row() and index (the last argument) have been updated to updatedIndex.